### ISSUE TITLE
Processing Plugin Error

### ISSUE THREAD

**Issue by pkparchure** [qgis/QGIS#42573]
IMPORTANT:

If the issue concerns a third party plugin (downloaded with the plugin manager) then it can't be fixed by the QGIS team.
Please raise your issue in the dedicated bug tracker for that specific plugin (as listed in the plugin's description)

Tickets created here will be closed.

**Comment by pkparchure** [comment]
I have Qgis instilled on my windows 10 system. The Processing Plugin of QGIS  started giving trouble after I installed Thonny IDE along with python 37. The error was.

"Couldn't load plugin 'processing' due to an error when calling its classFactory() method 
OSError: [WinError 193] %1 is not a valid Win32 application."

Since I had recently installed Thonny and installed several packages, I started removing the packages one by one.
Luckily after removing some packages I found that the culprit was the NumPy package. 
As soon as I un-installed NumPy from within Thonny, the Qgis Processing Plugin started working. 
Actually the python packages are installed by Thonny in 
C:\Users\xxxxx\AppData\Roaming\Python\Python37\site-packages
Now that is only temporary solution as I need to use NumPy within Thonny.

Any solution? Kindly let me know

Regards

Pradeep
